lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -254,6 +258,16 @@ def add_customer(post, options)
         post[:customer] = customer if customer.start_with?('cus_')
       end
 
+      def add_fulfillment_date(post, options)
+        post[:fulfillment_date] = options[:fulfillment_date] if options[:fulfillment_date]
```

Comment:
When core passes in fields for the options hash, they are made into strings. Does this field need to remain an integer when it gets passed to the gateway? If so, we may need to add a `.to_i` to it.

Reply:
Thank you, this is a good point.

The field supposedly accepts a timestamp. Here are my results from testing out various values:
* `1636756194` (Integer) - passes
* `'1636756194'` (String) - passes
* `'12'` (String, not a timestamp) - passes
* `0` (Integer) - fails with error `"Invalid timestamp: must be an integer"`
* `'whatever I want'` - fails with error `"Invalid integer: whatever I want"`

So it looks like the gateway is accepting any integer or any string that resolves to an integer, which seems to indicate that casting the value with `to_i` won't make a difference.

BUT, I'm going to do it anyways, just because we already know it's supposed to be a timestamp so I would rather make it an integer before we send it versus relying on the gateway's API to accept timestamps that are both string and integer, which is functionality we don't have control over and could change at any time.

Reply:
Very thorough testing, much appreciated. And I think your logic for making the change is a good one. Thanks!